datadog_checks_dev/datadog_checks/dev/tooling/commands/test.py

```diff
@@ -188,7 +190,7 @@ def test(
             enter_pdb=enter_pdb,
             debug=debug,
             bench=bench,
-            latest_metrics=latest_metrics,
+            latest_metrics=latest,
```

Comment:
Why not also change both to`latest`?
```suggestion
            latest=latest,
```